samples/core/GetStarted/Model.cs

```diff
@@ -11,7 +11,7 @@ public class BloggingContext : DbContext
         // The following configures EF to create a Sqlite database file as `C:\blogging.db`.
         // For Mac or Linux, change this to `/tmp/blogging.db` or any other absolute path.
         protected override void OnConfiguring(DbContextOptionsBuilder options)
-            => options.UseSqlite(@"Data Source=C:\blogging.db");
+            => options.UseSqlite(@"Data Source=blogging.db");
```

Comment:
I think this will cause much confusion about where the file is located when debugging